docs/general/administration/hardware-acceleration/intel.md

```diff
@@ -657,17 +657,34 @@ This has been tested with LXC 3.0 and may or may not work with older versions.
 
 #### LXC On Proxmox
 
-:::note
+1. Make sure your GPU is available as a DRI render device on the Proxmox host, e.g. `/dev/dri/renderD128`.
+   If not, [install the necessary drivers](./index.md#configure--verify-hardware-acceleration) on the host.
```

Comment:
link to the intel driver docs, not the general hardware acceleration docs

Reply:
fixed